index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="./style.css">
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css">
+
+    <style>
+        /* Stackoverflow preview fix, please ignore */
+        
+        .navbar-nav {
+            flex-direction: row;
+        }
+        
+        .nav-link {
+            padding-right: .5rem !important;
+            padding-left: .5rem !important;
+        }
+    </style>
```

Comment:
- it is okay to research online to resolve when you are stuck, but always try to understand the code and make it your own. 
- This style code should be moved to style.css to ensure easy reading and proper code structure.

Reply:
- [ ]  You have done great pointing out that the student needs to personalize the code.  Please remember to be polite and offer your opinion as a suggestion. The use of words like **Please, Kindly, I suggest** does the trick. something like _**Please always try to understand the code and make it your own.**_ is polite enough.

- [ ]  A good code review always explains why a change is requested. *example* Please move this CSS code to the style.css file.   Separating CSS from HTML keeps your code base clean and easy to maintain and update